Packs/Slack/ReleaseNotes/2_5_3.md

```diff
@@ -0,0 +1,3 @@
+#### Integrations
```

Comment:
Remove

Reply:
@amshamah419 why ? when I run _demisto-sdk validate -g_ or/and the one to create Release Note, it automatically add this one and i f you take a look to all previous Release Notes, they contain this format (see Release Notes 2_5_1, 2_5_0 etc)